src/css/content-selector-to-slotted.ts

```diff
@@ -66,11 +66,11 @@ class ContentToSlottedUsages extends CssRule {
           }
 
           warnings.push(new Warning({
-            code: 'content-to-slotted-usages',
+            code: 'content-selector-to-slotted',
             severity: Severity.WARNING, parsedDocument, sourceRange,
             message:
                 'The ::content pseudo-element has been deprecated in favor of ' +
-                'the ::shadow psuedo-element in WebComponents v1.',
+                'the ::slotted psuedo-element in WebComponents v1.',
```

Comment:
Last thing: WebComponents v1 -> ShadowDOM v1

Reply:
Oh, sorry, I missed that.